example/imports.yaml

```diff
@@ -16,6 +16,42 @@ virtualGarden:
   etcd:
     storageClassName: my-virtual-garden-storage-class
     handleETCDPersistentVolumes: true
+
+### backup settings for OSS (alicloud)
+#    backup:
+#      infrastructureProvider: alicloud
+#      region: oss-eu-central-1.aliyuncs.com
+#      bucketName: testvirtualgarden
+#      credentials:
+#        data:
+#          accessKeyID:     "your access key id"
+#          accessKeySecret: "your access key secret"
```

Comment:
I could be stated that the access credentials should not be base64 encoded.

Reply:
Same for all the other credentials

Reply:
Done